ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/daemon-terminattr.j2

```diff
@@ -1,75 +1,98 @@
-{%         if daemon_terminattr.smashexcludes is arista.avd.defined %}
-{%             set cvp_config.cli = cvp_config.cli ~ " -smashexcludes=" ~ daemon_terminattr.smashexcludes %}
-{%         else %}
-{%             set cvp_config.cli = cvp_config.cli ~ " -smashexcludes=" %}
-{%         endif %}
-{%         if daemon_terminattr.ingestexclude is arista.avd.defined %}
-{%             set cvp_config.cli = cvp_config.cli ~ " -ingestexclude=" ~ daemon_terminattr.ingestexclude %}
-{%         else %}
-{%             set cvp_config.cli = cvp_config.cli ~ " -ingestexclude=" %}
-{%         endif %}
-{%         if daemon_terminattr.ingestvrf is arista.avd.defined %}
-{%                 set cvp_config.cli = cvp_config.cli ~ " -ingestvrf=" ~ daemon_terminattr.ingestvrf %}
-{%         endif %}
-{%         if daemon_terminattr.disable_aaa is arista.avd.defined(true) %}
-{%             set cvp_config.cli = cvp_config.cli ~ " -disableaaa" %}
+{%     endif %}
+{%     if daemon_terminattr.cvaddrs is arista.avd.defined %}
+{%         set cvp_config.cli = cvp_config.cli ~ " -cvaddr=" ~ daemon_terminattr.cvaddrs | join(',') %}
+{%         if daemon_terminattr.cvauth.method == 'key' %}
+{%             set cvp_config.cli = cvp_config.cli ~ " -cvauth=key," ~ daemon_terminattr.cvauth.key %}
+{%         elif daemon_terminattr.cvauth.method == 'token' %}
+{%             set cvp_config.cli = cvp_config.cli ~ " -cvauth=token," ~ daemon_terminattr.cvauth.token_file %}
+{%         elif daemon_terminattr.cvauth.method == 'token-secure' %}
+{%             set cvp_config.cli = cvp_config.cli ~ " -cvauth=token-secure," ~ daemon_terminattr.cvauth.token_file %}
 {%         endif %}
+{%     endif %}
+{%     if daemon_terminattr.cvvrf is arista.avd.defined %}
+{%         set cvp_config.cli = cvp_config.cli ~ " -cvvrf=" ~ daemon_terminattr.cvvrf %}
+{%     endif%}
+{%     if daemon_terminattr.cvsourceip is arista.avd.defined %}
+{%         set cvp_config.cli = cvp_config.cli ~ " -cvsourceip=" ~ daemon_terminattr.cvsourceip %}
+{%     endif%}
+{%     if daemon_terminattr.cvgnmi is arista.avd.defined %}
+{%         set cvp_config.cli = cvp_config.cli ~ " -cvgnmi" %}
+{%     endif%}
+{%     if daemon_terminattr.cvobscurekeyfile is true %}
+{%         set cvp_config.cli = cvp_config.cli ~ " -cvobscurekeyfile" %}
+{%     endif%}
+{%     if daemon_terminattr.disable_aaa is true %}
+{%         set cvp_config.cli = cvp_config.cli ~ " -disableaaa" %}
+{%     endif%}
+{%     if daemon_terminattr.cvproxy is arista.avd.defined %}
+{%         set cvp_config.cli = cvp_config.cli ~ " -cvproxy=" ~ daemon_terminattr.cvproxy %}
+{%     endif%}
+{%     if daemon_terminattr.grpcaddr is arista.avd.defined %}
+{%         set cvp_config.cli = cvp_config.cli ~ " -grpcaddr=" ~ daemon_terminattr.grpcaddr %}
+{%     endif%}
+{%     if daemon_terminattr.grpcreadonly is true %}
+{%         set cvp_config.cli = cvp_config.cli ~ " -grpcreadonly" %}
+{%     endif%}
+{%     if daemon_terminattr.smashexcludes  %}
```

Comment:
please change to `{% if daemon_terminattr.smashexcludes is arista.avd.defined %}`
Same below